zebROS_ws/src/ros_control_boilerplate/src/candle_convert.cpp

```diff
@@ -0,0 +1,133 @@
+#include "ros_control_boilerplate/candle_convert.h"
+
+using namespace hardware_interface::candle;
+using namespace ctre::phoenix::led;
+
+namespace candle_convert {
+
+ColorFlowAnimation::Direction convertCANdleDirection(int direction) {
+    switch (direction) {
+        case 0:
+            return ColorFlowAnimation::Direction::Forward;
+        case 1:
+            return ColorFlowAnimation::Direction::Backward;
+    }
```

Comment:
Normally I like to thrown in a default case which prints an error. It very rarely pops up once code is working but does help catch weird cases during debugging.

Reply:
Added a default case